packages/react-components/react-dialog/etc/react-dialog.api.md

```diff
@@ -21,15 +21,15 @@ export const dialogClassName = "fui-Dialog";
 export const dialogClassNames: SlotClassNames<DialogSlots>;
 
 // @public
-export type DialogProps = ComponentProps<DialogSlots> & DialogCommons;
+export type DialogProps = ComponentProps<DialogSlots>;
 
 // @public (undocumented)
 export type DialogSlots = {
     root: Slot<'div'>;
 };
 
 // @public
-export type DialogState = ComponentState<DialogSlots> & DialogCommons;
+export type DialogState = ComponentState<DialogSlots> & Required<Pick<DialogProps, never>>;
```

Comment:
To avoid needing to build if the other comment is accepted.

```suggestion
export type DialogState = ComponentState<DialogSlots> & Required<DialogProps>;
```